templates/Dockerfile-deb.tmpl

```diff
@@ -37,6 +37,11 @@ RUN set -ex; \
     rm -rf "$GNUPGHOME"; \
     apt-key list
 
+# add buster backports repo to be able to download missing packages in buster main repo
+RUN ( [ %%os%% != debian ] || [ %%release%% != buster ] ) || ( \
+    echo "deb http://deb.debian.org/debian buster-backports main" >> /etc/apt/sources.list  \
+    )
```

Comment:
Note: Debian buster EOL'd on August 22'. It is bound to get more problematic to install packages in the future for this distro.

However I am ok to keep this as is, as the image builds fine for now apparently.